content/en/continuous_integration/setup_pipelines/custom_commands.md

```diff
@@ -37,30 +43,14 @@ datadog-ci trace [--name <name>] -- <command>
 
 Specify a valid [Datadog API key][2] in the `DATADOG_API_KEY` environment variable. For example:
 
-{{< site-region region="us" >}}
-{{< code-block lang="bash" >}}
-DATADOG_API_KEY=<api_key> datadog-ci trace \
-  --name "Greet" \
-  -- \
-  echo "Hello World"
-{{< /code-block >}}
-{{< /site-region >}}
-{{< site-region region="eu" >}}
-{{< code-block lang="bash" >}}
-DATADOG_API_KEY=<api_key> DATADOG_SITE=datadoghq.eu datadog-ci trace \
-  --name "Greet" \
-  -- \
-  echo "Hello World"
-{{< /code-block >}}
-{{< /site-region >}}
-{{< site-region region="us3" >}}
-{{< code-block lang="bash" >}}
-DATADOG_API_KEY=<api_key> DATADOG_SITE=us3.datadoghq.com datadog-ci trace \
+<pre class="chroma">
+<code class="language-bash" data-lang="bash">
+DATADOG_API_KEY=&lt;api_key&gt; DATADOG_SITE={{< region-param key="dd_site" >}} datadog-ci trace \
 --name "Greet" \
 -- \
 echo "Hello World"
-{{< /code-block >}}
-{{< /site-region >}}
+</code>
+</pre>
```

Comment:
@ruthnaebeck Is there a reason to use html instead of code-block here?

Reply:
The `{{< region-param key="dd_site" >}}` won't work inside the code-block partial or with backticks. This is the only way it would work.